worker/src/RTC/PlainTransport.cpp

```diff
@@ -592,6 +594,7 @@ namespace RTC
 							MS_THROW_TYPE_ERROR("missing port");
 						}
 
+						// NOLINTNEXTLINE (bugprone-unchecked-optional-access)
```

Comment:
Here I'm ignoring (bugprone-unchecked-optional-access) rule because  `port` is already defined as `uint16_t` and we cannot override it with an option value. Renaming it to another name does not feel good either. So here we're basically telling the static analyser that we know what we're doing.